source/install/requirements.rst

```diff
@@ -101,60 +101,31 @@ Hardware Sizing for Team Deployments
-- (Optional) Add one additional identical server for high availability mode, where one Mattermost server can be disabled or upgraded without interrupting service quality. Second server should be sized to carry the full load of the first server so performance does not degrade when the first server is taken offline.
-
-**Mattermost Server** (1 to 2 depending on level of redundancy and high availability required) 
-
-- One server with 4-8 vCPUs/cores, 16-32 GB RAM.
-- Minimum 4 GB SSD (solid state drive) storage for the binary and related files.
-- (Optional) Add one additional identical server for high availability mode, where one Mattermost server can be disabled or upgraded without interrupting service quality. Second server should be sized to carry the full load of the first server so performance does not degrade when the first server is taken offline. Note: The high availability option is available only by `contacting the Enterprise Edition team <https://about.mattermost.com/contact/>`_.
-
-**Network Attached Storage** 
-
-- One NAS server with 4-8 TB of storage (based on moderate storage of 10 MB per user per month times 20,000 users times 3 years of history, times 2x safety factor) or sized appropriately for your desired usage requirements. For high availability it is recommended you select a NAS server offering redundancy.
+For Enterprise Edition deployments with multi-server set up, see `our scaling guide <https://docs.mattermost.com/deployment/scaling.html>`_.
 
-**Database Server** (2 recommended for redundancy) 
+It is highly recommended that pilots are run before enterprise-wide deployments in order to estimate full scale usage based on your specific organizational needs. You can use the Mattermost open source load testing framework to simulate usage of your system: `https://github.com/mattermost/mattermost-load-test <https://github.com/mattermost/mattermost-load-test>`_.
 
-- One database server with 8-16 vCPUs/cores, 16-32 GB memory.
-- Minimum 100 GB SSD (solid state drive) storage for the binary and related files.
-- (Recommended) Add one identical database server to setup a Master-Slave configuration where the master can failover to slave with minimal disruption to service.
+Mattermost's `performance monitoring <https://docs.mattermost.com/deployment/metrics.html>`_ tools can be used for detailed performance measurements and inspect the running system to ensure sizing and installation is correct. 
 
-**Notes:**
+.. note::
 
-- Regular hard drives can be used in place of solid-state hard drives if having top performance is not a priority. If using a mix of HDD and SSD drives, the greatest performance gain would come from using SSD in the database server.
+  Regular hard drives (HDD) can be used in place of solid-state hard drives (SSD) if having top performance is not a priority. If using a mix of HDD and SSD drives, the greatest performance gain comes from using SSD in the database server.
```

Comment:
Is this note a duplication of https://github.com/mattermost/docs/pull/2302/files#diff-12b8342499d6b7515730e432a66bcda9R113

Reply:
Essentially